### ISSUE TITLE
Make BOTANY_TOURNAMENT_CLOSE_AT configurable from the environment

### ISSUE THREAD

**Issue by mjpieters** [inglesp/botany#25]
From server `settings.py` file:

```
# TODO read this from environment (and make optional)
BOTANY_TOURNAMENT_CLOSE_AT = datetime(2018, 9, 18, 14, 30, tzinfo=bst)
```

Development is much easier with this set based on a relative date or disabled. What env variable did you have in mind? Perhaps set a 'DEPLOYMENT` variable that defaults to 'production' and when set to 'development' instead the above is set to a relative date with an optional override?

**Comment by mjpieters** [comment]
For now, I just set this to `datetime.max`.

**Comment by dokato** [comment]
@mjpieters I'm taking care of it in #24 if you don't mind.

**Comment by inglesp** [comment]
Fixed in #34.
